New advisors (#756)
percona/pmm-doc: docs/how-to/configure.md

When active, PMM will automatically check for updates and put a notification in the home page *Updates* dashboard if any are available.

### Advisors

> Advisors are set of checks grouped by functionality that run a range of database health checks on a registered instance. 

Reviewer: ```suggestion
Advisors are a set of checks grouped by functionality that run a range of database health checks on a registered instance. 
```

> The findings are repored in the **Advisors > Failed Checks** page and an overview is displayed on the Dasboard, in the Failed Advisor Checks panel.  

Reviewer: ```suggestion
The findings are reported on the **Advisors > Failed Checks** page, and an overview is displayed on the Dashboard in the Failed Advisor Checks panel.  
```

By default, the Advisors option is disabled. 

> Checks are re-fetched and re-run at intervals. 

Reviewer: ```suggestion
Checks are refetched and rerun at intervals. 
```
